css/index.css

```
@@ -165,6 +167,36 @@ h3 {
   box-shadow: 0 0 5px 0 black;
 }
 
+/* projects section */
+#projects h1 {
+  padding-top: 200px;
+  margin-bottom: -2px;
+}
+
+#projects p {
+  padding-bottom: 20px;
+}
+
+#projects ul {
+  list-style: none;
+  margin-left: -40px;
+  display: flex;
+}
+
+#project li {
```

```suggestion
#projects li {
```

I think this is missing an "s"... or if you're happy with the display as-is, maybe this rule can be dropped?